frontend/src/components/settings/EditProfile.tsx

```diff
@@ -69,7 +92,7 @@ export function EditProfile() {
             onInput={onChangeField}
             rows={3}
             maxLength={150}
-            value={profileInfo.bio}
+            value={profileInfo.bio ? profileInfo.bio : ''}
```

Comment:
In this case, you can use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing).

You could also do this checking when setting the `profileInfo` state.